style/resources/cityscape.svg

```diff
@@ -0,0 +1,383 @@
+<svg 
```

Comment:
@Fredasaurus, regarding the new svg asset, I need 1 single separate cloud svg and 1 separate city svg.  The gradient and green bar are already in the chart, so do not need those.  Also if you want the colored flags to be svg I need those separate .svg files as well. (w/o the text)